pages/index.html

```diff
@@ -31,6 +31,8 @@
   <link rel="stylesheet" href="https://vyxal.github.io/Vylight/mode-vyxal.css">
 
   <script defer src="./vyxal.js"></script>
+  <script defer src="./dictionary.js"></script>
+  </script>
```

Comment:
```suggestion
```
Not sure if this is a stray closing script tag

Reply:
it is